share/migrations/0007_auto_20161122_1810.py

```diff
@@ -26,7 +26,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
```

Comment:
The choices are in a different order, because they are sorted now. If you run `makemigrations` over and over right now you'll get changes to types each time.
This mitigates that issue.